packages/contracts/src/ubiquistick/UbiquiStick.sol

```diff
@@ -120,17 +120,18 @@ contract UbiquiStick is
     function _beforeTokenTransfer(
         address from,
         address to,
-        uint256 tokenId
+        uint256 tokenId,
+        uint256 batchSize
     ) internal override(ERC721, ERC721Enumerable) {
-        super._beforeTokenTransfer(from, to, tokenId);
+        super._beforeTokenTransfer(from, to, tokenId, batchSize);
     }
 
     function _beforeConsecutiveTokenTransfer(
         address,
         address,
         uint256,
         uint96
-    ) internal override(ERC721, ERC721Enumerable) {
```

Comment:
why remove the override? @rndquu it conflicts with the other branch, i think this would trigger compiler warning

Reply:
> why remove the override? @rndquu it conflicts with the other branch, i think this would trigger compiler warning

it was not overriding anything

Reply:
let's retest, the idea is to avoid branching conflicts

Reply:
Error (4327): Function needs to specify overridden contracts "ERC721" and "ERC721Enumerable".
   --> src/ubiquistick/UbiquiStick.sol:128:5
   
   can you confirm. Not to dismiss your PR. i think its best to recheck!

@pavlovcik 
@rndquu 
this conflicted with #704

Reply:
<img width="517" alt="image" src="https://github.com/ubiquity/ubiquity-dollar/assets/125548724/3d4b7f76-238a-4631-ab23-f7f7fcd90bf8">

Reply:
it's not a good idea to rely on VSCode problems, the compiler shall tell you what's good and what's not

Reply:
<img width="713" alt="image" src="https://github.com/ubiquity/ubiquity-dollar/assets/125548724/826cadfe-70b4-4ed5-b77c-cf245255dbfd">

Reply:
compiler saying the same wym

Reply:
@molecula451 this PR updates dependencies so perhaps you should run `forge update` and check again